balena/models/device.py

```diff
@@ -174,6 +174,19 @@ def __set(
                     }
                 )
             else:
+                if len(uuid_or_id_or_ids) < 6:
+                    raise exceptions.AmbiguousDevice(uuid_or_id_or_ids)
```

Comment:
I guess an invalida parameter error might have been better, explaining that they need to provide at least 6 chars